Monitor and GPU changes

mkrgg opened the pull request:
**Monitor.xml**
- HDR Support in patent (need balance I-U-S). Need Graphics Card with HDR
- 2880p (5K) (need balance I-U-S) *
- HDMI
- DisplayPort
- Change G-SYNC to Adaptive Sync. Need DisplayPort
- Refresh Rate (30 Hz, 60 Hz, 120 Hz and 144 Hz) *

_*144 Hz and 2880p need to be linked to DisplayPort (Like in real life) but the design don't look nice, we need space :p_

Fix 1
- Added changes from last commit (780d0f7)
- Changed name: Adaptive Vsync to Adaptive Sync
- New desc: You can't have 60fps rock solid? No problem, Adaptive Sync is here!
- Re balanced Usability in HDR Support: 90 to 80
- Re balanced Usability in HBM1: 0 to 15
- Fixed the mistype: HBM2 to HBM1

**Graphics Card.xml**
- Change G-SYNC to Adaptive Sync.
- HDR Support in patent (need balance I-U-S)
- HBM1 and HBM2

Fix 1
- Added changes from last commit (bd3f351)
- Re balanced Usability in HDR Support: 90 to 80
- Changed name: Adaptive Vsync to Adaptive Sync
- Added Adaptive Sync to be a research from DisplayPort (Like IRL)

mkrgg committed:
Monitor and GPU changes

Monitor.xml
- HDR Support in patent (need balance I-U-S). Need Graphics Card with HDR
- 2880p (5K) (need balance I-U-S) *
- HDMI
- DisplayPort
- Change G-SYNC to Adaptive Sync. Need DisplayPort
- Refresh Rate (30 Hz, 60 Hz, 120 Hz and 144 Hz) *
*144 Hz and 2880p need to be linked to DisplayPort (Like in real life) but the design don't look nice, we need space :p

Graphics Card.xml
- Change G-SYNC to Adaptive Sync.
- HDR Support in patent (need balance I-U-S)
- HBM1 and HBM2

AlbinoGeek commented:
I apologize, changes I made today created some Merge Conflicts.

I'll look into if they're easy for me to fix up, otherwise please revise the Push Request to merge conflicts.

azuzazuz commented:
HBM2 has "From="HBM2"" thing which prevents tha mod iniatialization... did you tested the mod before making a request? please test if your mod changes are correct by creating folder in the game "mods" directory with the name like "HardwareTest" and put all the mod files (folders and files) there, then launch the game with that test mod enabled. please sorry if that sounds a bit rude :)

azuzazuz commented:
but i like the idea of upcomming memory type

mkrgg commented:
no problem, was a mistype xD. I'm gonna fix later :)

azuzazuz commented:
thats mine PC MASTERRACE joke there at monitor line 183 -_-, you violated the copyrights!!! kidding :)

mkrgg committed:
Fixes #1 (HBM2, Adaptive Sync, HDR)

- Added changes from last commit (780d0f7)
- Changed name: Adaptive Vsync to Adaptive Sync
- New desc: You can't have 60fps rock solid? No problem, Adaptive Sync is here!
- Re balanced Usability in HDR Support: 90 to 80
- Re balanced Usability in HBM1: 0 to 15
- Fixed the mistype: HBM2 to HBM1

mkrgg committed:
Fixes #1 (HDR, Adaptive Sync, DisplayPort)

- Added changes from last commit (bd3f351)
- Re balanced Usability in HDR Support: 90 to 80
- Changed name: Adaptive Vsync to Adaptive Sync
- Added Adaptive Sync to be a research from DisplayPort (Like IRL)

mkrgg committed:
Update from last commit (10-04)

mkrgg committed:
Update from last commit (10-04)

mkrgg committed:
Update from last commit (10-04)

AlbinoGeek commented:
I have merged these changes in with "pr/4" #7 ; GitHub for Desktop didn't allow me to merge directly.  Continue discussion in the new PR thread.
